Assets/MixedRealityToolkit.Providers/WindowsMixedReality/XRSDK/XRSDKWindowsMixedRealityUtilitiesProvider.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.WindowsMixedReality;
+using System;
+
+#if WMR_ENABLED
+using UnityEngine.XR.WindowsMR;
+#endif // WMR_ENABLED
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.WindowsMixedReality
+{
+    /// <summary>
+    /// An implementation of <see cref="Toolkit.WindowsMixedReality.IWindowsMixedRealityUtilitiesProvider"/> for Unity's XR SDK pipeline.
+    /// </summary>
+    public class XRSDKWindowsMixedRealityUtilitiesProvider : IWindowsMixedRealityUtilitiesProvider
```

Comment:
I'm also thinking about removing this concept and replacing it with concrete static classes, the existing one for the legacy XR pipeline and a new one for XR SDK. I'm not sure there's any overlap in MRTK code where a generic one is needed, but I'm not sure how this might be used by other devs.

@davidkline-ms

Reply:
sounds good. do you want to do that in a separate PR or this one?

Reply:
Separate PR